### ISSUE TITLE
Not phising

### ISSUE THREAD

**Issue by thomashandgraaf** [MetaMask/eth-phishing-detect#324]

**Comment by 409H** [comment]
Can you expand on this?

**Comment by thomashandgraaf** [comment]
Well I guess it was phising, can't see what link I reported as not phising. But I was trying to get on EtherDelta, but might've gone through a phising link of that address. Can you still see what link I reported as no-phising (while on hindsight now I think it is?) 

So that I won't confuse them anymore in the future, and bookmark the legit website. Because I did just get phished about 0.7ETH unfortunately.

**Comment by 409H** [comment]
No, we don't track you so we can't see what you tried to report.

In future, can you put the domain you're reporting in the body of the issue. Can you go through your browser history and see if it's there and report it?

**Comment by thomashandgraaf** [comment]
I just found it, but I don't see the difference between the original etherdelta.com and the one in my history (that's still marked as a phising website by you guys; but I believe it's right because I never has issues with EtherDelta until I ignored MetaMask's phising warning)

This is the link address that gives me a phising warning: https://www.etḥerdelta.com/

As of typing this right now, I just saw the difference. it's the small dot (.) below the h.
Really sneaky way to get caught. I will keep using MetaMask and never ignore the phising anymore. I wonder now how I got this link in the first place... I won't use a different one than my bookmark from now on either.
